source/ZividNETSamples.sln

```diff
@@ -31,80 +31,122 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CaptureFromFileVis3D", "App
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CaptureVis3D", "Applications\Basic\Visualization\CaptureVis3D\CaptureVis3D.csproj", "{68EB8781-17AA-42E9-9524-0C18A45278C3}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HandEyeCalibration", "Applications\Advanced\HandEyeCalibration\HandEyeCalibration.csproj", "{8ACFFEC6-34E5-4E14-B08C-60051B1CFD2F}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HandEyeCalibration", "Applications\Advanced\HandEyeCalibration\HandEyeCalibration\HandEyeCalibration.csproj", "{8ACFFEC6-34E5-4E14-B08C-60051B1CFD2F}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MultiCameraCalibration", "Applications\Advanced\MultiCameraCalibration\MultiCameraCalibration.csproj", "{26F708E4-218E-4F17-8989-F5C42796C932}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UtilizeEyeInHandCalibration", "Applications\Advanced\HandEyeCalibration\UtilizeEyeInHandCalibration\UtilizeEyeInHandCalibration.csproj", "{FAEF093E-9EEC-4CE5-87FE-D6FDB45CE700}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```

Comment:
Why is this added?

Reply:
I dont know

Reply:
Maybe it depends on how your Visual Studio is configured. Do you know @knatten

Reply:
> I dont know, but is is also included in the current one. So I'm assuming that is how it is suppose to be

What do you mean? I don't see "Any CPU" configurations anywhere on master?

Reply:
I was misunderstanding Satjas question. They are added because of my configurations in Visual Studio, updated it now to exclude the "Any CPU" stuff